### PR TITLE
Add a community events calendar to community page

### PR DESCRIPTION
**Pull request by travishathaway**
Fixes: #59

### CODE REVIEW

**Comment by kenodegard** [comment]
Hm, I was expecting a dedicated page for the calendar instead of hiding further down on the community page

Can we also modify it to use UTC time instead of Berlin time?

**Comment by jaimergp** [comment]
https://deploy-preview-147--conda-dot-org.netlify.app/community#events-calendar - LGTM!

Agreed on the UTC timezone.

**Comment by travishathaway** [comment]
@kenodegard,

I think putting this calendar on its own page is a good idea. In order to do this in the best possible way, I had to slight reorganize the website. The "community" page is now under the "learn" section. It still retains its top level navigation link though. The event calendar is underneath the "community" section.

I think this should make it easier for users to browse the content on the site in the future, and it's also more inline with what https://www.nebari.dev/ does, which  I like.

cc: @jaimergp

**Comment by travishathaway** [comment]
I'm also using the "redirects" plugin to handle redirecting site visitors to the correct page.

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default function Header() {
           management ecosystem for projects of any size and complexity.
         </p>
         <div className={styles.header_content_input}>
-          <a className="button button--primary button--lg col" href="/community">Learn More</a>
+          <a className="button button--primary button--lg col" href="/learn/community">Learn More</a>
```
Comment (jaimergp) [inline src/components/Header/index.jsx:15]: I think this needs to be reverted.

File: package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "2.3.1",
+    "@docusaurus/plugin-client-redirects": "^2.4.1",
```
Comment (jaimergp) [inline package.json:20]: Do we still need this? I guess it doesn't hurt but we are not using it anymore, right?

File: community/calendar.mdx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Events calendar
```
Comment (kenodegard) [inline community/calendar.mdx:1]:
```suggestion
# Events Calendar
```

File: package.json (hunk 2 above)
Comment (travishathaway) [reply package.json:20]: Nope. I'll take it out 👍.

File: src/components/Header/index.jsx (hunk 1 above)
Comment (travishathaway) [reply src/components/Header/index.jsx:15]: Good catch. Will do.

**Comment by travishathaway** [comment]
@jaimergp and @kenodegard,

This PR should be good to go.

**Review by kenodegard — APPROVED** [review]

**Comment by jaimergp** [comment]
I am afraid `ctz=local` does not work as expected (I see times as in LA) and we will have to resort to something like [this SO answer proposes](https://stackoverflow.com/questions/31821974/support-user-time-zone-in-embedded-google-calendar).

**Comment by travishathaway** [comment]
Thanks for reporting that @jaimergp. We also realized that the time zones are not working. Unfortunately, a solution like like the one mentioned in that SO article would be pretty tricky to implement.

**Comment by travishathaway** [comment]
Maybe a stop-gap measure would be to just present it in a single timezone (UTC?) and then make sure to mention that above the calendar?

**Comment by jaimergp** [comment]
+1 for UTC as default.

**Pull request merged**